rir/src/R/Printing.h

```diff
@@ -8,7 +8,11 @@ namespace rir {
 
 static std::string dumpSexp(SEXP src, size_t length = 50) {
     CaptureOut rec;
-    Rf_PrintValue(src);
+    if (src == R_UnboundValue) {
+        std::cout << "R_UnboundValue";
```

Comment:
I think I liked the "-" better... Because in pir generated code we have a lot of them!